addon/templates/components/single-event.hbs

```diff
@@ -89,33 +94,31 @@
     <div class="single-event-learningmaterial-list">
       <h2>{{t "general.sessionLearningMaterials"}}</h2>
       {{single-event-learningmaterial-list
-        learningMaterials=sessionLearningMaterials
-      }}
+        learningMaterials=sessionLearningMaterials}}
```

Comment:
Ember eslint used to prevent this, maybe they changed it, but we've got `}}` on it's own line everywhere else in a multi-line component. @stopfstedt how do you feel about this stylistically? Keep it on it's own line, or we don't care?

Reply:
i don't care one way or the other. _whatever eslint says it should be._

Reply:
I used to have a new line for it then I started doing this style b/c it looked cleaner. If it's not an accepted thing here, I can revert and note this going forward.

Reply:
Let's go with this for now, and I'll add linting and clean up everything later. It seems like we have a mix of both.